dist/overview-style.css

```diff
@@ -192,10 +202,21 @@ ul.style-list {
 li.style-list-item {
   display: flex;
   margin: 10%;
+  /* setting position property to checkmark can position based on this and not the body */
+  position: relative;
+  opacity: 50%;
+}
+
+svg.style-list-item-checkmark {
+  position: absolute;
+  right: 0;
+  border-radius: 50%;
+  background:rgba(255, 255, 255, 0.6)
 }
 
 span.style-list-item-circle {
```

Comment:
just curious, did you want these thumbnails to have this much opacity all the time?
<img width="328" alt="Screen Shot 2022-04-13 at 3 02 43 PM" src="https://user-images.githubusercontent.com/51180529/163278921-10e2f05a-5918-4307-ade5-9efbd34478a2.png">